ramls/accountdata.json

```diff
@@ -38,17 +38,9 @@
       ]
     },
     "paymentStatus": {
-      "description": "Overall status of the payment/waive/transfer/refund/cancel",
-      "type": "object",
-      "properties": {
-        "name": {
-          "description": "Name of the status (values used are Outstanding, Paid partially, Paid fully, Waived partially, Waived fully, Transferred partially, Transferred fully, Refunded partially, Refunded fully, Cancelled as error)",
-          "type": "string"
-        }
-      },
-      "required": [
-        "name"
-      ]
+      "description": "Overall status of the payment/waive/transfer/refund/credit/cancel",
+      "type": "string",
```

Comment:
We can't change the type from object to string. This is a breaking change - it will break the UI and all of the modules that create accounts. The reason is that previously this field was serialized to JSON like this:
```
"paymentStatus": {
  "name": "Paid fully"
}
```
but after your change it will look like this
```
"paymentStatus": "Paid fully"
```
you shouldn't change `paymentStatus`'es type from object to string, you should make a `name` field of the `paymentStatus` an enum.